dev/Materials/Reveal/RevealBrush_rs2_themeresources.xaml

```diff
@@ -2603,9 +2603,129 @@
     </Style>
 
     <Style TargetType="ListViewItem" x:Key="ListViewItemRevealBackgroundShowsAboveContentStyle" />
-    
+
     <Style TargetType="GridViewItem" x:Key="GridViewItemRevealBackgroundShowsAboveContentStyle" />
 
+    <Style TargetType="controls:DropDownButton" x:Key="DropDownButtonRevealStyle" >
```

Comment:
What was this style/template copied from? ButtonRevealStyle plus some changes?

Reply:
It is a mix of the ButtonRevealStyle and the DropDownButtomStyle template.

Reply:
I did a diff and looks like it's DropDownButtonStyle + reveal background and border changes + reveal state setters. Looks good to me : )